backend/main/server/app.go

```diff
@@ -1095,6 +1089,8 @@ func (a *App) updateCommunity(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	payload.Name = &c.Name
```

Comment:
i'm assuming `Name` is the only field not being submitted to the API via the FE?

Reply:
yes exactly, that's why we assign it here